src/software/ai/hl/stp/stp.h

```diff
@@ -122,22 +124,36 @@ class STP : public HL
     PlayInfo getPlayInfo() override;
 
     /**
-     * Given a list of tactics and the current World, assigns robots from the friendly
-     * team to each tactic
+     * Given a vector of a vector of tactics and the current World, assigns robots
+     * from the friendly team to each tactic
      *
      * Some tactics may not be assigned a robot, depending on if there is a robot
      * capable of performing that tactic
      *
      * This will clear all assigned robots from all tactics
      *
-     * The order of the given tactics determines their priority, with the tactics as the
-     * beginning of the vector being a higher priority than those at the end. The priority
-     * determines which tactics will NOT be assigned if there are not enough robots on the
-     * field to assign them all. For example, if a Play returned 6 Tactics but there were
-     * only 4 robots on the field at the time, only the first 4 Tactics in the vector
-     * would be assigned to robots and run.
+     * The outer vector ranks the inner vector of tactics by priority. Tactics in
+     * lower indexes of the outer vector will be assigned first. For example:
      *
-     * @param tactics The list of tactics that should be assigned a robot. Note
+     * {
+     *      {goalie_tactic},
+     *      {crease_defender_1, crease_defender_2},
+     *      {move_tactic},
+     * }
+     *
+     * The cost of assigning a goalie_tactic will be minimized across all robots first,
+     * followed by both the crease_defender tactics. The move_tactic will be assigned
+     * last.
+     *
+     * The order of the given tactics in the inner vector also determines their priority,
+     * with the tactics at the beginning of the vector being a higher priority than those
+     * at the end. The priority determines which tactics will NOT be assigned if there are
+     * not enough robots on the field to assign them all. For example, if a Play returned
+     * 4 Tactics in total but there were only 3 robots on the field at the time, only the
+     * first 3 Tactics in the vectors would be assigned to robots and run. (In the example
+     * above, only the goalie and crease_defenders would be assigned)
+     *
+     * @param tactics The list of list of tactics that should be assigned a robot. Note
```

Comment:
```suggestion
     * @param tactics The vector of vector of tactics that should be assigned a robot. Note
```
Can this be renamed for consistency?

Reply:
fixed